testdata/test.proto

```diff
@@ -0,0 +1,105 @@
+syntax = "proto3";
```

Comment:
you can update the existing example instead of creating new

Reply:
Daniel-garcia suggested to add test case.
I added a test case in a similar manner done in atlas-gentool.

Reply:
Can you point me to test cases ?

Reply:
test case added in testdata folder.